root/static/scripts/edit/URLCleanup.js

```diff
@@ -2023,12 +2023,15 @@ const CLEANUPS = {
     match: [new RegExp('^(https?://)?(www\\.)?operabase\\.com', 'i')],
     type: LINK_TYPES.otherdatabases,
     clean: function (url) {
-      return url.replace(/^(?:https?:\/\/)?(?:www\.)?operabase\.com\/a\/([^\/?#]+)\/([0-9]+).*$/, 'https://operabase.com/a/$1/$2');
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?operabase\.com\//, 'https://operabase.com/');
+      url = url.replace(/^https:\/\/operabase\.com\/a\/[^\/?#]+\/([0-9]+).*$/, 'https://operabase.com/artists/$1');
+      url = url.replace(/^https:\/\/operabase\.com\/artists\/[^0-9]+([0-9]+).*$/, 'https://operabase.com/artists/$1');
+      return url;
```

Comment:
It can still call `replace` only once as before:

```suggestion
      return url.replace(/^(?:https?:\/\/)?(?:www\.)?operabase\.com\/(?:a\/[^\/?#]+|artists)\/([0-9]+).*$/, 'https://operabase.com/artists/$1');
```

Reply:
I guess! It feels a bit less obvious but I guess regexes aren't too obvious anyway.